File: packages/live-update/ios/Plugin/Classes/Results/GetCustomIdResult.swift
import Foundation
import Capacitor

@objc public class GetCustomIdResult: NSObject, Result {
    let customId: String?

    init(customId: String?) {
        self.customId = customId
    }

    public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["customId"] = customId == nil ? NSNull() : customId
        return result as AnyObject
    }
}

File: packages/live-update/android/src/main/java/io/capawesome/capacitorjs/plugins/liveupdate/classes/results/GetCustomIdResult.java
package io.capawesome.capacitorjs.plugins.liveupdate.classes.results;

import androidx.annotation.Nullable;
import com.getcapacitor.JSObject;
import io.capawesome.capacitorjs.plugins.liveupdate.interfaces.Result;
import org.json.JSONObject;

public class GetCustomIdResult implements Result {

    @Nullable
    private String customId;

    public GetCustomIdResult(@Nullable String customId) {
        this.customId = customId;
    }

    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("customId", customId == null ? JSONObject.NULL : customId);
        return result;
    }
}
